### PR TITLE
Filipino Translation - 1 129 translated words

### PR DESCRIPTION
**Pull request by mind-blast**
Revision for files components.rst, middleware.rst and pages-controller.rst as per project owners request.
Translated request-response.rst with 1 129 translation files.
Note :  Code text and computer generated code text are not included as translated words.

### CODE REVIEW

File: tl/controllers/pages-controller.rst [hunk 1]
```diff
@@ -1,17 +1,17 @@
-The Pages Controller
-####################
+Ang mga Pahina sa Controller
```
Comment (josteinfrancois) [inline tl/controllers/pages-controller.rst]: `Ang mga Pahina na Controller`

File: tl/controllers/pages-controller.rst [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+############################
 
-CakePHP's official skeleton app ships with a default controller **PagesController.php**.
-This is a simple and optional controller for serving up static content. The home page
-you see after installation is generated using this controller and the view
-file **src/Template/Pages/home.ctp**. If you make the view file
-**src/Template/Pages/about_us.ctp** you can access it using the URL
-**http://example.com/pages/about_us**. You are free to modify the Pages
-Controller to meet your needs.
+Ang CakePHP na opisyal na balangkas na app ay nagpapadala namay default na controller **PagesController.php**.
```
Comment (josteinfrancois) [inline tl/controllers/pages-controller.rst]: - `ay pinapadala na may default na controller na **PagesController.php**.`

File: tl/controllers/pages-controller.rst [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+Ito ay simple at opsyonal na controller para paghahatid ng static na nilalaman. Ang home page
+na iyong nakikita pagkatapos sa installation ay nabuo sa paggammit ng controller na ito at ang tiningnan na
```
Comment (josteinfrancois) [inline tl/controllers/pages-controller.rst]: I think you should retain the word `view` here: `at ang view na file`

File: tl/controllers/pages-controller.rst [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
+file **src/Template/Pages/home.ctp**. Kung ikaw ay makagawa ng tiningnan na file
```
Comment (josteinfrancois) [inline tl/controllers/pages-controller.rst]: - `file na **src/Template/Pages/home.ctp**`
- `view na file`

File: tl/controllers/pages-controller.rst [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
+**src/Template/Pages/about_us.ctp** maaari kang maka-access nito gamit ang URL
```
Comment (josteinfrancois) [inline tl/controllers/pages-controller.rst]: - `file na **src/Template/Pages/about_us.ctp**`

File: tl/controllers/pages-controller.rst [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
+**http://example.com/pages/about_us**. Malaya ka na makapagbago sa Mga Pahina
```
Comment (josteinfrancois) [inline tl/controllers/pages-controller.rst]: - `gamit ang URL na **http://example.com/pages/about_us**`

File: tl/controllers/pages-controller.rst [hunk 7]
```diff
@@ -1,17 +1,17 @@
...
+ng Controller upang matugunan ang iyang mga pangangailangan.
```
Comment (josteinfrancois) [inline tl/controllers/pages-controller.rst]: - `Mga Pahina na Controller`
- `ang iyong mga pangangailangan.`

File: tl/controllers/pages-controller.rst [hunk 8]
```diff
@@ -1,17 +1,17 @@
...
 
-When you "bake" an app using Composer the Pages Controller is created in your
-**src/Controller/** folder.
+Kapag ikaw ay "nag-bake" ng isang app gamit ang Composer ng mga Pahina sa Controller ay nilikha sa iyong
```
Comment (josteinfrancois) [inline tl/controllers/pages-controller.rst]: - `ang mga Pahina na Controller`

File: tl/controllers/pages-controller.rst [hunk 9]
```diff
@@ -1,17 +1,17 @@
...
+**src/Controller/** na folder.
 
 .. meta::
-    :title lang=en: The Pages Controller
-    :keywords lang=en: pages controller,default controller,cakephp,ships,php,file folder,home page
+    :title lang=en: Ang mga Pahina sa Controller
```
Comment (josteinfrancois) [inline tl/controllers/pages-controller.rst]: `Pahina na Controller`

File: tl/controllers/pages-controller.rst [hunk 10]
```diff
@@ -1,17 +1,17 @@
...
+    :keywords lang=en: mga pahina sa controller,default na controller,cakephp,mga pagpapadala,php,file folder,home page
```
Comment (josteinfrancois) [inline tl/controllers/pages-controller.rst]: `mga pahina na controller`

File: tl/controllers/request-response.rst [hunk 11]
```diff
@@ -1,296 +1,296 @@
-Request & Response Objects
+Hiling & Tugon na mga Object
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Hiling at Tugon`

File: tl/controllers/request-response.rst [hunk 12]
```diff
@@ -1,296 +1,296 @@
...
 ##########################
 
 .. php:namespace:: Cake\Http
 
-The request and response objects provide an abstraction around HTTP requests and
-responses. The request object in CakePHP allows you to introspect an incoming
-request, while the response object allows you to effortlessly create HTTP
-responses from your controllers.
+Ang hiling at tugon ng mga object ay magbugay ng isang abstraction na umiikot sa HTTP na mga hiling at
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: - `na mga object`
- `ay nagbibigay`
- `ng isang abstraksyon`

File: tl/controllers/request-response.rst [hunk 13]
```diff
@@ -1,296 +1,296 @@
...
-* Providing access to request parameters both as array indexes and object
-  properties.
-
-As of 3.4.0, CakePHP's request object implements the `PSR-7
-ServerRequestInterface <http://www.php-fig.org/psr/psr-7/>`_ making it easier to
-use libraries from outside of CakePHP.
-
-Request Parameters
-------------------
+Ang ``ServerRequest`` ay isang default na hiling na object na ginamit sa CakePHP. Nagtatayo ito ng
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Nagsasagitna ito ng ilang mga tampok`

File: tl/controllers/request-response.rst [hunk 14]
```diff
@@ -1,296 +1,296 @@
...
+numero ng mga tampok para magtanong at makigpag-ugnayan sa hiniling na datos.
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `makipag-ugnayan`

File: tl/controllers/request-response.rst [hunk 15]
```diff
@@ -1,296 +1,296 @@
...
+Bawat isa na hiling sa isang Request ay nalikha at pagkatapos napasa sa reperensiya sa
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Bawat isang kahilingan`

File: tl/controllers/request-response.rst [hunk 16]
```diff
@@ -1,296 +1,296 @@
...
-
-Request Parameters
-------------------
+Ang ``ServerRequest`` ay isang default na hiling na object na ginamit sa CakePHP. Nagtatayo ito ng
+numero ng mga tampok para magtanong at makigpag-ugnayan sa hiniling na datos.
+Bawat isa na hiling sa isang Request ay nalikha at pagkatapos napasa sa reperensiya sa 
+iba-ibang mga layer ng isang aplikasyon na gumagamit ng hiling na datos. Bilang default ang hiling
+ay nakatalaga sa ``$this->request``, at magagamit sa mga Controller, mga Cell, mga View
+at mga Helper. Maaari mo ring ma-access ito sa mga Component na gumagamit ng controller
+na reperensiya. Ang ilang mga tungkulin ``ServerRequest`` ay nagsasagawa ng isinama:
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: ```` ng ilang mga tungkulin na isinasagawa ng ``ServerRequest`` ay nagsasama ng:````

File: tl/controllers/request-response.rst [hunk 17]
```diff
@@ -1,296 +1,296 @@
...
+
+* Ang pagproseso sa GET, POST, at mga FILE na mga array sa istraktura ng datos na ikaw ay
+  pamilyar.
+* Pagbibigay ng environment introspeksyon na nauukol sa hiling. Ang impormasyon
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `environment na introspeksyon `

File: tl/controllers/request-response.rst [hunk 18]
```diff
@@ -1,296 +1,296 @@
...
+  na tulad ng mga header na pinadala, ang IP address ng kliyente, at ang subdomain/domain
+  na mga pangalan sa server ng iyong aplikasyon na pinatakbo.
+* Pagbibigay ng access sa hiling na mga parameter sa parehong bilang ng array na mga index at object
+  na mga katangian.
+
+Tulad ng 3.4.0, ang CakePHP na hiling na object na nagpapatupad sa `PSR-7
+ServerRequestInterface <http://www.php-fig.org/psr/psr-7/>`_ ginagawang mas madali ang
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `na ginagawang `

File: tl/controllers/request-response.rst [hunk 19]
```diff
@@ -1,296 +1,296 @@
...
+paggamit ng mga library mula sa labas sa CakePHP.
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `labas ng CakePHP`

File: tl/controllers/request-response.rst [hunk 20]
```diff
@@ -1,296 +1,296 @@
...
-are several important/useful parameters that CakePHP uses internally, these
-are also all found in the routing parameters:
+Ang lahat ay magkakaloob sa iyo ng access upang maipasa ang mga argumento. Doon
+ay may ilang importante/kapaki-pakinabang na mga parameter na ang CakePHP ay gumagamit sa panloob, ang mga ito
+ay natagpuan din ang lahat sa routing na mga parameter:
 
-* ``plugin`` The plugin handling the request. Will be null when there is no
+* ``plugin`` Ang plugin na paghahawak ng hiling. Ay magiging null kapag walang
   plugin.
 * ``controller`` The controller handling the current request.
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: Please translate this one

File: tl/controllers/request-response.rst [hunk 21]
```diff
@@ -1,296 +1,296 @@
...
-* ``action`` The action handling the current request.
-* ``prefix`` The prefix for the current action. See :ref:`prefix-routing` for
-  more information.
+* ``action`` Ang aksyon na paghahawak ay kasalukuyang hiling.
+* ``prefix`` Ang prefix para sa kasalukuyang aksyon. Tingnan :ref:`prefix-routing` para sa
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Tingnan ang`

File: tl/controllers/request-response.rst [hunk 22]
```diff
@@ -1,296 +1,296 @@
...
     $page = $this->request->getQuery('page');
 
-    // Prior to 3.4.0
+    // Bago ang 3.4.0
     $page = $this->request->query('page');
 
-You can either directly access the query property, or you can use
-``getQuery()`` method to read the URL query array in an error-free manner.
-Any keys that do not exist will return ``null``::
+Maaari kang magdirekta na mag-access sa query ng katangian, o maaari kang makagamit sa
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: - `Maaari kang direktang mag-access`
- `query na katangian`

File: tl/controllers/request-response.rst [hunk 23]
```diff
@@ -1,296 +1,296 @@
...
+``getQuery()`` na pamamaraan upang basahin ang URL query na array sa isang pagkamali-libreng paraan.
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `pagkamali-libreng` change to `walang pagkakamali na`

File: tl/controllers/request-response.rst [hunk 24]
```diff
@@ -1,296 +1,296 @@
...
+Anumang mga susi na hindi umiiral ay babalik sa ``null``::
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: Should we translate `key`?

File: tl/controllers/request-response.rst [hunk 25]
```diff
@@ -1,296 +1,296 @@
...
 
     $foo = $this->request->getQuery('value_that_does_not_exist');
     // $foo === null
 
-    // You can also provide default values
+    // Maaari mo ring magbigay ng default na mga halaga
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Maaari ka ring magbigay`

File: tl/controllers/request-response.rst [hunk 26]
```diff
@@ -1,296 +1,296 @@
...
     $foo = $this->request->getQuery('does_not_exist', 'default val');
 
-If you want to access all the query parameters you can use
+Kung ikaw ay gustong mag-access sa lahat ng query na mga parameter ay maaari kang gumamit sa
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Kung gusto mong ma-access ang lahat ng query na mga parameter maaari kang gumamit ng`

File: tl/controllers/request-response.rst [hunk 27]
```diff
@@ -1,296 +1,296 @@
...
+-----------------------
 
 .. php:method:: getData($name, $default = null)
 
-All POST data can be accessed using
-:php:meth:`Cake\\Http\\ServerRequest::getData()`.  Any form data that
-contains a ``data`` prefix will have that data prefix removed. For example::
+Lahat ng POST na datos ay maaaring ma-access gamit ang
+:php:meth:`Cake\\Http\\ServerRequest::getData()`.  Anumang porma ng datos na
+naglalaman ng ``data`` na prefix ay magkakaroon ng datos prefix na inalis. Halimbawa::
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: ` ``data`` na prefix ay tatanggalan ng prefix na ito.`

File: tl/controllers/request-response.rst [hunk 28]
```diff
@@ -1,296 +1,296 @@
...
 
-    // An input with a name attribute equal to 'MyModel[title]' is accessible at
+    // Ang isang input na may isang pangalan na katangian na pantay sa 'MyModel[title]' ay naa-access sa 
     $title = $this->request->getData('MyModel.title');
 
-Any keys that do not exist will return ``null``::
+Anumang susi na hindi umiiral ay ibabalik ang ``null``::
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `ay magbabalik ng ``null`` `

File: tl/controllers/request-response.rst [hunk 29]
```diff
@@ -1,296 +1,296 @@
...
+ay awtomatikong ma-parse at maitakda sa ``$this->data`` para sa ``PUT`` at
+``DELETE`` na mga hiling. Kung ikaw ay tumatanggap ng JSON o XML na datos, tingnan sa ibaba para sa kung paano
+ikaw maka-access sa mga hiling na katawan na iyon.
 
-When accessing the input data, you can decode it with an optional function.
-This is useful when interacting with XML or JSON request body content.
-Additional parameters for the decoding function can be passed as arguments to
+Kapag nag-access sa input na datos, maaari kang maka-decode nito na may isang opsyonal na function.
+Ito ay kapaki-pakinabang kapag nakipag-ugnayan sa XML o JSON na hiling sa buong nilalaman.
+Karagdagang mga parameter para sa pag-decode ng function ay maaaring mapasa bilang mga argumento sa
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Ang karagdagang`

File: tl/controllers/request-response.rst [hunk 30]
```diff
@@ -1,296 +1,296 @@
...
-Environment Variables (from $_SERVER and $_ENV)
------------------------------------------------
+Environment na mga Variable (from $_SERVER and $_ENV)
+-----------------------------------------------------
 
 .. php:method:: env($key, $value = null)
 
-``ServerRequest::env()`` is a wrapper for ``env()`` global function and acts as
-a getter/setter for enviromnent variables without having to modify globals
+``ServerRequest::env()`` ay isang tagapagbalot para sa ``env()`` sa global punsyon at gumaganap bilang
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: - `Ang ``ServerRequest::env()`` `
- `global na punsyon`

File: tl/controllers/request-response.rst [hunk 31]
```diff
@@ -1,296 +1,296 @@
...
-non-URL-encoded post bodies. You can read input data in any format using
-:php:meth:`~Cake\\Http\\ServerRequest::input()`. By providing a decoding function,
-you can receive the content in a deserialized format::
+Ang mga aplikasyon ay gumagamit ng :doc:`/development/rest` nang madalas na pagpapalit ng datos sa
+non-URL-encoded post na mga body. Maaari kang bumasa ng input na datos sa anumang format gamit ang
+:php:meth:`~Cake\\Http\\ServerRequest::input()`. Sa pamamagitan ng pagbibigay ng pag-decode na function,
+maaari kang makakuha sa nilalaman sa isang deserialized na format::
 
-    // Get JSON encoded data submitted to a PUT/POST action
+    // Kunin ang JSON na naka-encod na datos na sinumete sa isang PUT/POST na aksyon
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `naka-encode`

File: tl/controllers/request-response.rst [hunk 32]
```diff
@@ -1,296 +1,296 @@
...
-the 'as array' parameter on ``json_decode``. If you want XML converted into a
-DOMDocument object, :php:meth:`~Cake\\Http\\ServerRequest::input()` supports
-passing in additional parameters as well::
+Ang ilang deserializing na mga pamamaraan ay kailangan ng karagdagang mga parameter kapag natawag, tulad ng
+isang 'as array' na parameter sa ``json_decode``. Kung gusto mo ang XML na naka-convert sa isang
+DOMDocument na object, :php:meth:`~Cake\\Http\\ServerRequest::input()` na sumusuporta
+sa pagpasa sa karagdagang na mga parameter din::
 
-    // Get XML encoded data submitted to a PUT/POST action
+    // Kunin ang XML na naka-encod na datos na sinumete sa isang PUT/POST na aksyon
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `naka-encode`

File: tl/controllers/request-response.rst [hunk 33]
```diff
@@ -1,296 +1,296 @@
...
     $data = $this->request->input('Cake\Utility\Xml::build', ['return' => 'domdocument']);
 
-Path Information
-----------------
+Path ng Impormasyon
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Path na Impormasyon`

File: tl/controllers/request-response.rst [hunk 34]
```diff
@@ -1,296 +1,296 @@
...
+-------------------
 
-The request object also provides useful information about the paths in your
-application. The ``base`` and ``webroot`` attributes are useful for
-generating URLs, and determining whether or not your application is in a
-subdirectory. The attributes you can use are::
+Ang hiling na object ay nagbibigay din ng kapaki-pakinabang na impormasyon tungkol sa iyong mga path sa iyong
+aplikasyon. Ang ``base`` at ``webroot`` na mga katangian ay kapaki-pakinabang para sa
+pagbubuo ng mga URL, at pagtukoy kung o hindi ang iyong aplikasyon ay nasa isang
+subdirektoryo. Ang mga katangian na maaari mong gamitin ang::
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `maaari mong gamitin ay::`

File: tl/controllers/request-response.rst [hunk 35]
```diff
@@ -1,296 +1,296 @@
...
 
-    // Assume the current request URL is /subdir/articles/edit/1?page=1
+    // Ipagpalagay ang kasalukuyang hiling na URL sa /subdir/articles/edit/1?page=1
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `na URL ay`

File: tl/controllers/request-response.rst [hunk 36]
```diff
@@ -1,296 +1,296 @@
...
----------------------------
+Pagsusuri ng Hiling na mga Kondisyon
+------------------------------------
 
 .. php:method:: is($type, $args...)
 
-The request object provides an easy way to inspect certain conditions in a given
-request. By using the ``is()`` method you can check a number of common
-conditions, as well as inspect other application specific request criteria::
+Ang hiling na object ay nagbibigay ng isang madali na paraan para tingnan ang mga kondisyon sa binigay na
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `madaling paraan`

File: tl/controllers/request-response.rst [hunk 37]
```diff
@@ -1,296 +1,296 @@
...
-  create a regular expression. Subsequent calls to add an already defined
-  options detector will merge the options.
-* Callback detectors - Callback detectors allow you to provide a 'callback' type
-  to handle the check. The callback will receive the request object as its only
-  parameter.
+Maaari mo ring palawakin ang hiling na mga detektor na magagamit, sa pamamagitan sa paggamit ng
+:php:meth:`Cake\\Http\\ServerRequest::addDetector()` upang lumikha ng bagong mga uri ng
+mga detektor. Mayroong apat na magkaibang mga uri ng mga detektor na maaari kang lumikha:
+
+* Environment na halaga ng paghahambing - ay naghahambing sa halaga ng nakuha mula sa :php:func:`env()`
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Ang Environment na halaga na paghahambing`

File: tl/controllers/request-response.rst [hunk 38]
```diff
@@ -1,296 +1,296 @@
...
+  para sa pagkakapantay-pantay na may binigay na halaga.
+* Pattern na halaga ng paghahambing - Ang pattern na halaga ng paghahambing ay nagpapahintulot sa iyo na maghambing sa
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Ang Pattern na halaga na paghahambing`

File: tl/controllers/request-response.rst [hunk 39]
```diff
@@ -1,296 +1,296 @@
...
+  halaga na nakuha mula sa :php:func:`env()` sa isang regular na ekspresyon.
+* Pagpipilian batay sa paghahambing -  Ang opsyon ay nagbabase sa mga paghahambing sa paggamit ng isang listahan ng mga opsyon upang
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Nakabase sa Opsyon na paghahambing`

File: tl/controllers/request-response.rst [hunk 40]
```diff
@@ -1,296 +1,296 @@
...
 
-    // Add an option detector
+    // Magdagdag ng opsyon na detektor
     $this->request->addDetector('internalIp', [
         'env' => 'CLIENT_IP',
         'options' => ['192.168.0.101', '192.168.0.100']
     ]);
 
-    // Add a callback detector. Must be a valid callable.
+    // Magdagdag ng callback na detektor. Kailangang isang balido na matawagan.
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `matatawagan`

File: tl/controllers/request-response.rst [hunk 41]
```diff
@@ -1,296 +1,296 @@
...
     $this->request->addDetector(
         'awesome',
         function ($request) {
             return $request->getParam('awesome');
         }
     );
 
-    // Add a detector that uses additional arguments. As of 3.3.0
+    // Magdagdag ng isang dektektor na gumagamit sa karagdagan na mga argumento. Bilang ng 3.3.0
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Batay sa 3.3.0`

File: tl/controllers/request-response.rst [hunk 42]
```diff
@@ -1,296 +1,296 @@
...
+Mayroong ilang mga built-in na mga detektor na magagamit mo:
+
+* ``is('get')`` Suriin upang makita kung ang kasalukuyang hiling ay isang GET.
+* ``is('put')`` Suriin upang makita kung ang kasalukuyang  hiling ay isang PUT.
+* ``is('patch')`` Suriin upang makita kung ang kasalukuyang hiling ay isang PATCH.
+* ``is('post')`` Suriin upang makita kung ang kasalukuyang hiling ay isang POST.
+* ``is('delete')`` Suriin upang makita kung ang kasalukuyang hiling ay isang DELETE.
+* ``is('head')`` Suriin upang makita kung ang kasalukuyang hiling ay HEAD.
+* ``is('options')`` Suriin upang makita kung ang kasalukuyang hiling ay OPTIONS.
+* ``is('ajax')`` Suriin upang makita kung ang kasalukuyang hiling nanggaling na may
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `hiling na darating na may`

File: tl/controllers/request-response.rst [hunk 43]
```diff
@@ -1,296 +1,296 @@
...
   X-Requested-With = XMLHttpRequest.
-* ``is('ssl')`` Check to see whether the request is via SSL.
-* ``is('flash')`` Check to see whether the request has a User-Agent of Flash.
-* ``is('requested')`` Check to see whether the request has a query param
-  'requested' with value 1.
-* ``is('json')`` Check to see whether the request has 'json' extension and
-  accept 'application/json' mimetype.
-* ``is('xml')`` Check to see whether the request has 'xml' extension and accept
-  'application/xml' or 'text/xml' mimetype.
+* ``is('ssl')`` Suriin upang makita kung ang kasalukuyang  hiling ay dumadaan sa SSL.
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: - `kung ang hiling ay sa pamamagitan ng SSL`
- remove `kasalukuyang`

File: tl/controllers/request-response.rst [hunk 44]
```diff
@@ -1,296 +1,296 @@
...
+* ``is('flash')`` Suriin upang makita kung ang kasalukuyang hiling ay mayroong isang User-Agent ng Flash.
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: remove `kasalukuyang`

File: tl/controllers/request-response.rst [hunk 45]
```diff
@@ -1,296 +1,296 @@
...
+* ``is('requested')`` Suriin upang makita ang kasalukuyang hiling ay mayroong isang query param
+  'requested' na may halaga na 1.
+* ``is('json')`` Suriin upang makita ang kasalukuyang hiling ay mayroong 'json' na ekstensyon at
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: remove `kasalukuyang`

File: tl/controllers/request-response.rst [hunk 46]
```diff
@@ -1,296 +1,296 @@
...
+  tumatanggap ng 'application/json' na mimetype.
+* ``is('xml')`` Suriin upang makita ang kasalukuyang hiling ay mayroong 'xml' na ekstensyon at tumatanggap ng
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: remove `kasalukuyang`

File: tl/controllers/request-response.rst [hunk 47]
```diff
@@ -1,296 +1,296 @@
...
+  'application/xml' o 'text/xml' na mimetype.
 
 .. versionadded:: 3.3.0
-    Detectors can take additional parameters as of 3.3.0.
+    Mga Detektor ay maaaring tumatagal ng karagdagang mga parameter sa bilang ng 3.3.0.
```
Comment (josteinfrancois) [inline tl/controllers/request-response.rst]: `Ang mga Detektor ay maaaring tumanggap ng karagdagang mga parameter batay sa 3.3.0.`

**Review by josteinfrancois — COMMENTED** [review]
Hey @robin-ho. I have finished reviewing your work. Please see the comments. Thank you!

File: tl/controllers/pages-controller.rst (hunk 1 above)
Comment (mind-blast) [reply tl/controllers/pages-controller.rst]: done chnage

File: tl/controllers/pages-controller.rst (hunk 2 above)
Comment (mind-blast) [reply tl/controllers/pages-controller.rst]: done change

File: tl/controllers/pages-controller.rst (hunk 3 above)
Comment (mind-blast) [reply tl/controllers/pages-controller.rst]: done change.

File: tl/controllers/pages-controller.rst (hunk 4 above)
Comment (mind-blast) [reply tl/controllers/pages-controller.rst]: done change

File: tl/controllers/pages-controller.rst (hunk 5 above)
Comment (mind-blast) [reply tl/controllers/pages-controller.rst]: done change

File: tl/controllers/pages-controller.rst (hunk 6 above)
Comment (mind-blast) [reply tl/controllers/pages-controller.rst]: done change

File: tl/controllers/pages-controller.rst (hunk 7 above)
Comment (mind-blast) [reply tl/controllers/pages-controller.rst]: done change

File: tl/controllers/pages-controller.rst (hunk 8 above)
Comment (mind-blast) [reply tl/controllers/pages-controller.rst]: done change

File: tl/controllers/pages-controller.rst (hunk 9 above)
Comment (mind-blast) [reply tl/controllers/pages-controller.rst]: done change

File: tl/controllers/pages-controller.rst (hunk 10 above)
Comment (mind-blast) [reply tl/controllers/pages-controller.rst]: done change

File: tl/controllers/request-response.rst (hunk 11 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 12 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 13 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 14 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 15 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 16 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 17 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 18 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 19 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 20 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 21 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 22 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 23 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 24 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: i think it is better to retain. change back susi to key

File: tl/controllers/request-response.rst (hunk 25 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 26 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 27 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 28 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 29 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 30 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 31 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 32 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 33 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 34 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 35 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 36 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 37 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 38 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 39 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 40 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 41 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 42 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 43 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 44 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 45 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 46 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

File: tl/controllers/request-response.rst (hunk 47 above)
Comment (mind-blast) [reply tl/controllers/request-response.rst]: done change

**Comment by mind-blast** [comment]
I have changed and committed as per suggested words.
Thank you for the proof read. =)

**Pull request merged**